wordclock/wificfg.c

```diff
@@ -1302,18 +1303,6 @@ void wificfg_init(uint32_t port, const wificfg_dispatch *dispatch)
 
 		xTaskCreate(dns_task, "WiFi Cfg DNS", 512, NULL, 2, NULL);
```

Comment:
die DNS task mag uit. Die is nodig voor captive DNS. Heb het idee dat hij nu aleen maar kwaad kan.